src/main/java/org/jasig/portlet/newsreader/adapter/RomeAdapter.java

```diff
@@ -31,8 +31,8 @@
 import net.sf.ehcache.Element;
 import org.apache.commons.io.IOUtils;
 import org.apache.commons.lang.StringUtils;
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Cool.  Good to know about the tool.  I remember seeing it but hadn't thought to look into it when I had converted other portlets.  At least it gets us 80% of the way there!  (the other 20% being the cleanup of using {} and sometimes removing the isDebugEnabled where appropriate).  Sounds like a useful tool to run on uPortal and the portlets at some point when we can handle a massive set of changes.